app/controllers/api/v2/compute_resources_controller.rb

```diff
@@ -223,13 +223,39 @@ def associate
         if @compute_resource.supports_host_association?
           associator = ComputeResourceHostAssociator.new(@compute_resource)
           associator.associate_hosts
-          @hosts = associator.hosts
-          render 'api/v2/hosts/index', :layout => 'api/v2/layouts/index_layout'
+          handle_errors(associator)
```

Comment:
Is removing `render 'api/v2/hosts/index'` intentional? I thought it will be rendered in case of success, but after this change it seems that only errors will be rendered. How does this behave in case of successful association?